packages/core-modules/test/contracts/modules/ElectionModule.resolve.test.js

```diff
@@ -33,50 +44,202 @@ describe('ElectionModule (resolve)', () => {
       );
     });
 
-    describe('when entering the evaluation period', function () {
+    describe('when entering the nomiantion period', function () {
       before('fast forward', async function () {
-        await fastForwardTo(await ElectionModule.getEpochEndDate(), ethers.provider);
+        await fastForwardTo(await ElectionModule.getNominationPeriodStartDate(), ethers.provider);
       });
 
-      it('shows that the current period is Evaluation', async function () {
-        assertBn.eq(await ElectionModule.getCurrentPeriodType(), ElectionPeriod.Evaluation);
+      before('nominate', async function () {
+        await ElectionModule.connect(candidate1).nominate();
+        await ElectionModule.connect(candidate2).nominate();
+        await ElectionModule.connect(candidate3).nominate();
+        await ElectionModule.connect(candidate4).nominate();
+        await ElectionModule.connect(candidate5).nominate();
       });
 
-      describe('before evaluating the epoch', function () {
-        describe('when trying to resolve the epoch', function () {
-          it('reverts', async function () {
-            await assertRevert(ElectionModule.resolve(), 'EpochNotEvaluated');
-          });
-        });
+      it('shows that nominations exist', async function () {
+        assertBn.eq((await ElectionModule.getNominees()).length, 5);
       });
 
-      describe('when evaluating the epoch', function () {
-        before('evaluate', async function () {
-          await ElectionModule.evaluate();
+      describe('when entering the election period', function () {
```

Comment:
add a test that consider the issue in _updateWinnerSet(). It can be simulated with 3 ballots and 2 seats. 
You need to get the ballots to check get in that order (Candidate1: 2 votes, Candidate2: 1 vote, Candidate3: 10 votes).  
Candidate3 will replace Candidate1 => the council will end as C3, C2 while C1 received more votes than C2

Reply:
Also done.